build.assets/Dockerfile-bpf

```diff
@@ -0,0 +1,26 @@
+# Minmal container fo generating BPF bytecode
```

Comment:
```suggestion
# syntax=docker/dockerfile:1

# Minimal container for generating BPF bytecode
```

Reply:
Do we need this header even if I'm not using any "extended" docker syntax?

Reply:
Nope, but just for consistency among all dockerfiles.

But there is a typo fix as well -- s/Minmal/Minimal/